tests/installation/bootloader_s390.pm

```diff
@@ -298,7 +298,7 @@ sub run {
     my $exception = $@;
 
     # Add y2start.log for output
-    die join("\n", '##########', `ls -l /var/log/ 2>&1 && cat /var/log/YaST2/vncserver.log`) if $exception;
+    die join("\n", '##########', `ls -l /var/log/YaST2/ 2>&1 && cat /var/log/YaST2/vncserver.log`) if $exception;
```

Comment:
i would suggest the same format as L302. `'#' x <multiplier>` if you dont mind

Reply:
Thank you for the suggestion. I had to debug, why the log files did not exist for a while. I have fixed that now.